Kickstarter-iOS/Views/Cells/CommentPostFailedCell.swift

```diff
@@ -86,6 +86,21 @@ final class CommentPostFailedCell: UITableViewCell, ValueCell {
         _ = self.retryButton
           |> status == .retrying ? postingButtonStyle : retryButtonStyle
       }
+
+    self.viewModel.outputs.shouldIndentContent
+      .observeForUI()
+      .observeValues { shouldIndent in
+        _ = self.rootStackView
+          |> \.axis .~ .vertical
```

Comment:
Should we save this as a style constant somewhere globally and use it both here and in the `CommentCell`/wherever else we use `shouldIntentContent`?

Reply:
yea that would be ideal. Right now though I only see it being used for threaded replies, but if another place in the app is using this kind of indentation we can more it into `CommentCellStyles`.